README.md

```diff
@@ -77,11 +77,9 @@ The current corresponding IDE versions of these aliases can be found [here](./in
 We welcome contributions to support new IDE versions. However, to make
 the review process faster and easier, we recommend the following:
 
-  * We can only accept small pull requests. A smaller pull request would likely have less review
-    comments and hence can get submitted much faster. For us, it is also much easier to integrate
-    such a small, focused pull request as it likely conflicts less with our internal code base. 
-    For example, you should have separate pull requests each focusing on a certain incompatible change
-    rather than having a large pull request fixing multiple ones.
```

Comment:
Sorry for not being precise with my previous comment. I unfortunately keep forgetting that GitHub doesn't show a range selection with the comments. I thought that we could keep the "For example, you should have ..." part and only suggested a slightly different formulation for the previous sentences. Thanks for updating the formulation. Would you want to add the "For example, you should have ..." sentence again? I'll leave it up to you whether you want to add it or directly merge this pull request.